### PR TITLE
fix(links): fix links that were valid in the previous doc

### PR DESCRIPTION
**Pull request by educhastenier**
before the new Doc infra & tooling refactoring

### CODE REVIEW

File: modules/ROOT/pages/single-sign-on-with-kerberos.adoc [hunk 1]
```diff
@@ -208,7 +208,7 @@ edit the authenticationManager-config.properties as follows:
      -->	logout.link.hidden=true
 ----
 +
-Make sure to link:multi-tenancy-and-tenant-configuration#toc2[set the right tenant admin username].
+Make sure to link:multi-tenancy-and-tenant-configuration[set the right tenant admin username].
```
Comment (educhastenier) [inline modules/ROOT/pages/single-sign-on-with-kerberos.adoc:212]: This paragraph does not exist anymore, as the multi-tenancy is now deprecated. I am not sure of what to do with this link

File: modules/ROOT/pages/single-sign-on-with-saml.adoc [hunk 2]
```diff
@@ -93,7 +93,7 @@ edit the authenticationManager-config.properties as follows:
         logout.link.hidden=true
 ----
 +
-Make sure to link:multi-tenancy-and-tenant-configuration#toc2[set the right tenant admin username].
+Make sure to link:multi-tenancy-and-tenant-configuration[set the right tenant admin username].
```
Comment (educhastenier) [inline modules/ROOT/pages/single-sign-on-with-saml.adoc]: This paragraph does not exist anymore, as the multi-tenancy is now deprecated. I am not sure of what to do with this link

**Comment by tbouffard** [comment]
ℹ️ This link containing `toc` anchor are probably wrong in 7.9+ branches as well.

**Comment by educhastenier** [comment]
>  This link containing `toc` anchor are probably wrong in 7.9+ branches as well.

Yes, for some of them, but from one branch to another, some more were added, so to speed up the fix, I did it on the 2021.1 branch only.
Another reason is that I found this bug when working on another subject, so I did not want to dedicate too much time to it.

However, if you think this is important to fix it for all 7.9+, I can take time to do it properly.

**Comment by educhastenier** [comment]
Ok, I did it on 7.9 as well: https://github.com/bonitasoft/bonita-doc/pull/1589

Not sure I will do it also on 7.10, 7.11, though

**Pull request merged**